cmd/mayactl/app/command/commands.go

```diff
@@ -35,7 +35,7 @@ func NewMayaCommand() *cobra.Command {
 				mapiserver.Initialize()
 				if mapiserver.GetConnectionStatus() != "running" {
 					fmt.Println("Unable to connect to mapi server address")
-					os.Exit(1)
+					//os.Exit(1)
```

Comment:
why this change?

Reply:
Updated the PR description and added the code comment too.